tests/contrib/langchain/test_langchain_community.py

```diff
@@ -1100,6 +1101,114 @@ def test_embedding_logs_when_response_not_completed(
+
+    model = chat_openai.configurable_alternatives(
+        langchain_core.runnables.ConfigurableField(id="model"),
+        default_key="chat_openai",
+        openai=openai,
+    )
+
+    chain = (
+        {
+            "topic": langchain_core.runnables.RunnablePassthrough(),
+            "style": langchain_core.runnables.RunnablePassthrough(),
+        }
+        | prompt
+        | model
+        | langchain_core.output_parsers.StrOutputParser()
+    )
+
+    with request_vcr.use_cassette("lcel_openai_chain_call_complicated.yaml"):
+        chain.invoke({"topic": "chickens", "style": "a 90s rapper"})
+
+
+@pytest.mark.asyncio
+@pytest.mark.snapshot
+async def test_lcel_chain_simple_async(langchain_core, langchain_openai, request_vcr):
+    prompt = langchain_core.prompts.ChatPromptTemplate.from_messages(
+        [("system", "You are world class technical documentation writer."), ("user", "{input}")]
+    )
+    llm = langchain_openai.OpenAI()
+
+    chain = prompt | llm
+    with request_vcr.use_cassette("lcel_openai_chain_acall.yaml"):
+        await chain.ainvoke({"input": "how can langsmith help with testing?"})
+
+
+@pytest.mark.snapshot
+def test_lcel_chain_batch(langchain_core, langchain_openai, request_vcr):
+    """
+    Test that invoking a chain with a batch of inputs will result in a 4-span trace,
+    with a root RunnableSequence span, then tree LangChain ChatOpenAI spans underneath
+    """
+    prompt = langchain_core.prompts.ChatPromptTemplate.from_template("Tell me a short joke about {topic}")
+    output_parser = langchain_core.output_parsers.StrOutputParser()
+    model = langchain_openai.ChatOpenAI()
+    chain = {"topic": langchain_core.runnables.RunnablePassthrough()} | prompt | model | output_parser
+
+    with request_vcr.use_cassette("lcel_openai_chain_batch.yaml"):
+        chain.batch(["chickens", "cows", "pigs"])
+
+
+@pytest.mark.snapshot
+def test_lcel_chain_nested(langchain_core, langchain_openai, request_vcr):
+    """
+    Test that invoking a nested chain will result in a 4-span trace with a root
+    RunnableSequence span, then another RunnubaleSequence and LangChain ChatOpenAI call,
+    and finally a second LangChain ChatOpenAI call.
+    """
+    prompt1 = langchain_core.prompts.ChatPromptTemplate.from_template("what is the city {person} is from?")
+    prompt2 = langchain_core.prompts.ChatPromptTemplate.from_template(
+        "what country is the city {city} in? respond in {language}"
+    )
+
+    model = langchain_openai.ChatOpenAI()
+
+    chain1 = prompt1 | model | langchain_core.output_parsers.StrOutputParser()
+    chain2 = prompt2 | model | langchain_core.output_parsers.StrOutputParser()
+
+    complete_chain = {"city": chain1, "language": itemgetter("language")} | chain2
+
+    with request_vcr.use_cassette("lcel_openai_chain_nested.yaml"):
+        complete_chain.invoke({"person": "Spongebob Squarepants", "language": "Spanish"})
+
+
+# @pytest.mark.asyncio
+# @pytest.mark.snapshot
```

Comment:
```suggestion
@pytest.mark.asyncio
@pytest.mark.snapshot
```
Looks like this test wasn't running